api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -2771,6 +2771,14 @@
             <type>ActivationFile</type>
           </association>
         </field>
+        <field>
+          <name>packaging</name>
+          <version>4.1.0+</version>
+          <description>Specifies that this profile will be activated based on the project's packaging.</description>
+          <association>
+            <type>ActivationPackaging</type>
```

Comment:
Do we need a next type with only one field?

Reply:
Oh, are you suggesting to add a single new field `packaging`, same as the existing `jdk` one ?